src/test/java/teammates/test/cases/logic/FeedbackResponsesLogicTest.java

```diff
@@ -51,6 +53,7 @@ public static void classSetUp() throws Exception {
         turnLoggingUp(FeedbackResponsesLogic.class);
         removeAndRestoreTypicalDataInDatastore();
         removeAndRestoreDatastoreFromJson("/SpecialCharacterTest.json");
+        removeAndRestoreDatastoreFromJson("/FeedbackSessionQuestionTypeTest.json");
```

Comment:
Will using this json affect other tests using it?
If it won't, can merge this PR.